compiler/test/suites/provides.re

```diff
@@ -220,32 +220,12 @@ describe("provides", ({test, testSkip}) => {
   );
   assertHasWasmExport(
     "issue_1872_reprovide_from_submodule",
-    "module Test; module M { provide let x = 1; provide let y = 2 }; use M.*; provide { x, y }",
+    "module Test; module M { provide let x = () => 1; provide let y = () => 2 }; use M.*; provide { x, y }",
     [
-      ("GRAIN$EXPORT$x", Binaryen.Export.external_global),
-      ("GRAIN$EXPORT$y", Binaryen.Export.external_global),
+      ("x", Binaryen.Export.external_function),
+      ("y", Binaryen.Export.external_function),
     ],
   );
-  assertHasWasmExport(
```

Comment:
If I understood the changes correctly we no longer allow you to export values right, which is why these tests are being removed ?

Reply:
There is no change to the final wasm output. The old wasm object files had these exports but the final linked wasm files did not.